app/controllers/katello/api/v2/alternate_content_sources_controller.rb

```diff
@@ -14,6 +14,9 @@ class Api::V2::AlternateContentSourcesController < Api::V2::ApiController
       param :description, String, desc: N_("Description for the alternate content source"), required: false
       param :base_url, String, desc: N_('Base URL for finding alternate content'), required: false
       param :subpaths, Array, desc: N_('Path suffixes for finding alternate content'), required: false
+      param :deb_releases, String, desc: N_('Debian releases/distributions (e.g., bookworm, bullseye)'), required: false
```

Comment:
```suggestion
      param :deb_releases, String, desc: N_('Debian releases/distributions (e.g., trixie, bookworm)'), required: false
```

suggestion to use latest two codenames as example.